modules/getting-started-openshift-gitops.adoc

```diff
@@ -0,0 +1,128 @@
+//
+// * openshift-docs/cicd/gitops/working-with-openshift-gitops.adoc
+
+[id="getting-started-with-red-hat-openshift-gitops_{context}"]
+= Getting Started with Red Hat OpenShift GitOps
+
+[id="installing-red-hat-openshift-gitops_{context}"]
+== Installing Red Hat OpenShift GitOps
+
+.Prerequisite
+
+The Red Hat product version of Argo CD is the Openshift GitOps Operator. When you install the Red Hat OpenShift GitOps operator, you get an instance of Argo CD in the 'openshift-gitops' namespace.
+
+To ensure you can work with the Red Hat OpenShift GitOps operator, go to xref:../operators/admin/olm-adding-operators-to-cluster.adoc#adding-operators-to-a-cluster[installing an operator from OperatorHub].
+
+After you install an Operator, you have a Red Hat OpenShift Container cluster. Continue with the default operator installation because this operator installs to all namespaces on the cluster. Now you have installed a Red Hat OpenShift GitOps operator.
+
+[id="logging-into-argocd_{context}"]
+== Logging into ArgoCD
+
+The Red Hat OpenShift GitOps operator automatically sets up a ArgoCD cluster configuration. ArgoCD manages specific cluster-scoped resources, including: platform operators, optional Operator Lifecycle Manager (OLM) operators, and user management. Because ArgoCD is not a cluster-admin, you must launch into the ArgoCD instance from the Console Application Launcher.
+
+To find the password for the ArgoCD:
+
+. Go to the table of contents and make sure you are in the *Developer* perspective and not the *Administrator* perspective.
+  *  You now see a list of available projects.
+. Select the *Secrets* tab.
+. Select the *argo-cluster-cluster* instance to reveal the password.
+. Copy the password.
+. Use this password to log into the ArgoCD UI.
+
+*Note:* if you want to fetch the ArgoCD password from the command line interface, run the following command:
+
+ `oc get secret argocd-cluster-cluster -n openshift-gitops -o jsonpath='{.data.admin\.password}' | base64 -d`
+
+[id="setting-up-a-new-argocd-instance_{context}"]
+== Setting up a new ArgoCD instance
+
+The Red Hat OpenShift GitOps operator automatically installs an instance of ArgoCD in the "openshift-gitops" namespace with additional permissions, allowing ArgoCD to manage cluster-scoped resources.
+
+If you want to install your own instance for managing your cluster configurations or deploying your applications, you must deploy your own instance of ArgoCD. Using your own instance of ArgoCD helps you to continuously deliver your applications.
+
+.Installing ArgoCD
+
+To install an ArgoCD instance:
+
+. Go to the table of contents and make sure you are in the *Administrator* perspective and not the *Developer* perspective.
+. Select *Operators* then select *Installed Operators*.
+. Create a new project or select an existing project, depending on where you want to install the ArgoCD instance.
+. Select *OpenShift GitOps* from the "Installed Operators" page.
```

Comment:
All GUI elements in bold (single asterix in asciidoc. The comments here use markdown so will render the single asterix into italics). Location, then action: In the *Installed Operators* page, select *OpenShift GitOps*